manifests/charts/istio-control/istio-discovery/templates/revision-tags.yaml

```diff
@@ -0,0 +1,58 @@
+{{- range $tagName := $.Values.revisionTags }}
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: istio-revision-tag-{{ $tagName }}
```

Comment:
do we need the namespace appended for non istio-system cases? I am not even sure it works, but the main one does it...

like 
```
{{- if eq $.Release.Namespace "istio-system"}}
  name: istio-sidecar-injector-{{ $tag }}
{{ else }}
  name: istio-sidecar-injector-{{ $tag }}-{{ $.Release.Namespace }}
{{- end }}
```